cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/InactiveOperationInfo.java

```diff
@@ -21,11 +21,14 @@
 
 import java.util.List;
 
-final class InactiveOperationInfo extends OperationInfo {
+/**
+ * OperationInfo representing an inactive operation.
+ */
+public final class InactiveOperationInfo extends OperationInfo {
```

Comment:
Actually, if you test using a query as discussed below then you will not need these changes.